manager/assets/modext/widgets/security/modx.panel.access.policy.js

```diff
@@ -128,6 +124,8 @@ Ext.extend(MODx.panel.AccessPolicy,MODx.FormPanel,{
             var r = this.config.record;
 
             this.getForm().setValues(r);
+            Ext.getCmp('modx-policy-header').html = Ext.util.Format.htmlEncode(r.name);
```

Comment:
This is incorrect, should be: `Ext.getCmp("modx-header-breadcrumbs").updateHeader(Ext.util.Format.htmlEncode(r.name));`